WordPress/gutenberg
# GradientPicker: Hard deprecate outer margins

Part of #39358

## What?

Promotes `__nextHasNoMargin` on GradientPicker to the default behavior.

## Why?

The deprecation grace period has elapsed.

## Testing Instructions

In the Storybook for GradientPicker, turn on the [margin checker tool](https://github.com/WordPress/gutenberg/assets/555336/f4fcf334-7c27-47ad-acba-e9b3a5f6f734) in the toolbar. The margin-free styles should now be the default, and the `__nextHasNoMargin` prop should not be listed in the props table.

✅ There is one direct usage of GradientPicker in the Gutenberg app, and it's already using it in margin-free mode.
✅ All internal components that use GradientPicker under the hood are already using it in margin-free mode.

## ✍️ Dev Note

See Dev Note for #58699.
